api-doc/openapi.yaml

```diff
@@ -89,7 +89,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/GameData'
+              $ref: '#/components/schemas/GameDataWithoutId'
```

Comment:
I'd rename this to something like GameDataInput, i.e. describe intent/usecase and not what it contains. If we for example have a new field in the output in the future, that is not required in the input, the name is out of date.

Reply:
Updated schema names, still not perfect but maybe little better